test/integration/020-create.js

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {api} = require('bedrock-payment-paypal');
+const {util} = require('bedrock');
+const {Errors} = require('bedrock-payment');
+
+const {BedrockError} = util;
+const minute = 60000;
+const twoMinutes = minute * 2;
```

Comment:
Maybe overkill for these testing cases, but the `ms` package makes this easier like `.timeout(ms('2m'))`.

Reply:
a bit of an overkill for test code.